metadata.rb

```diff
@@ -23,5 +23,15 @@
 end
 
 depends    'chef_handler', '>= 1.2'
```

Comment:
@jeffbyrnes shouldn't we also exclude `chef_handler` as it's in the Chef boneyard and raises Chef-33 deprecation error?